.travis.yml

```diff
@@ -16,4 +16,4 @@ script:
 
 notifications:
   email:
-    on_success: never
```

Comment:
Was this a line ending change?

Reply:
Yes, it's the result of an editor setting to accommodate other tools I use that end files with newlines, but looking into it more (https://github.com/atom/atom/issues/4741#issuecomment-114356248), it looks like it's a POSIX standard. Tools like `wc` depend on it to give a correct line count, so I'm going to say it's a good thing.

Reply:
SGTM